src/escalus_ct.erl

```diff
@@ -15,9 +15,6 @@
          get_config/1,
          log_stanza/3]).
 
-%% What about that?
--export([rpc_call/6]).
```

Comment:
``erlang
rpc_call(Mod, Fun, Args, #{timeout => 5000, node => cool_node, cookie => do_we_need_cookie_here})
```

Reply:
Because I can remember first 3 params. But not the rest. And usually the rest is good enough.

Reply:
Actually, I also tried something similar to test the idea out:

```erlang
call(RPCSpec, Mod, Fun, Args) -> ...
```

I think the above is slightly better than your suggestion, since it's a simple path from `distributed_helper:rpc/4`:

```erlang
rpc(Node, M, F, A) ->
```

to

```erlang
rpc(#{} = RPCSpec, M, F, A) ->
```

I mean it's easy to refactor big tests using `sed` and the like if we follow the pattern I suggest. One step at a time, though, let's leave it for another PR.